addons/binding/org.openhab.binding.harmonyhub/src/main/java/org/openhab/binding/harmonyhub/handler/HarmonyDeviceHandler.java

```diff
@@ -102,12 +102,13 @@ public void handleCommand(ChannelUID channelUID, Command command) {
     public void initialize() {
         id = getConfig().as(HarmonyDeviceConfig.class).id;
         name = getConfig().as(HarmonyDeviceConfig.class).name;
-        if (name == null && id <= 0) {
+        if (!checkConfig()) {
             logger.error("A harmony device thing must be configured with a device name OR a postive device id");
```

Comment:
don't log any error here, but add the message to the updateStatus call as a third parameter please!